### ISSUE TITLE
Legends don't render

### ISSUE THREAD

**Issue by GabrielSiq** [CodeYellowBV/chartist-plugin-legend#41]
Hi,

first of all, thanks for your work with this plugin.

This issue isn't a problem with the plugin but rather with my lack of experience using it. 

I didn't install the plugin via `npm`. Rather, I just copied the `.js` file to my `static` directory (it's a Flask app) and imported in my view.
```html
<script src="{{ url_for('static', filename='assets/js/chartist-plugin-legend.js')}}"></script>
<script src="{{ url_for('static', filename='assets/js/dashboard-update.js')}}"></script>
```
where dashboard-update contains the functions that request new data and update my line charts.

I currently initialize empty Chartist line charts when the page loads:
```javascript
 var data = {
                        labels: [],
                        series:[[]]
                    };
var options = {
                lineSmooth: Chartist.Interpolation.none({
                    fillHoles: true
                }),
                plugins: [
                        Chartist.plugins.legend({
                            legendNames: []
                        })
                    ]
            };
new Chartist.Line('.ct-chart', data, options);
```
And then update them via ajax:
```javascript
$.ajax({
    type: "POST",
    url: "/getChartData",
    contentType:"application/json",
    data : JSON.stringify(requestJSON),
    success: function(resultJSON) {
        if($.trim(resultJSON)) {
            var result = JSON.parse(resultJSON);
            var data = {
                labels: result['labels'],
                series: result['series']
            };
            var options = {
                lineSmooth: Chartist.Interpolation.none({
                    fillHoles: true
                }),
                plugins: [
                        Chartist.plugins.legend({
                            legendNames: legends
                        })
                    ]
            };
            chart.get(0).__chartist__.update(data, options);
        }
    },
    error: function() {
        ...
    }
});
```
where legends is an array with the same number of strings as there are series. However, at each update, new series are included, the chart itself loads, but the legends don't. 
![alt-text](http://i.imgur.com/dGvZonN.png)
How am I doing wrong?
Thanks!

**Comment by GabrielSiq** [comment]
Update: The issue seems to be with `Chartist`, that doesn't render the plugins on `.update()` and not with `chartist-plugin-legend`.

**Comment by SpaceK33z** [comment]
Hi, this plugin doesn't work with `.update()` yet. There is a PR open that fixes this, #40. It would be nice if you could test that PR and report your findings there, that would speed up the process!

**Comment by benmcm1994** [comment]
Any updates on this? Need to update the legend dynamically. Can I just delete the chart and make another instead of updating?
